src/Http/Response/Factory.php

```diff
@@ -7,8 +7,8 @@
 use Illuminate\Support\Str;
 use Dingo\Api\Http\Response;
 use Illuminate\Support\Collection;
-use Illuminate\Pagination\Paginator;
 use Dingo\Api\Transformer\Factory as TransformerFactory;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
```

Comment:
Thinking about it again.. maybe Fractal needs two illuminate paginator adapters. One for each paginator type, then at least we could get fractal to work in either case